src/AprilTagNode.cpp

```diff
@@ -215,6 +216,7 @@ void AprilTagNode::onCamera(const sensor_msgs::msg::Image::ConstSharedPtr& msg_i
         // 3D orientation and position
         geometry_msgs::msg::TransformStamped tf;
         tf.header = msg_img->header;
+        tf.header.frame_id = parent_frame_id;
```

Comment:
This will override the frame ID from the camera in which frame the tag is detected and there is no way to prevent this.